src/main/java/com/google/jenkins/plugins/computeengine/client/ComputeClientV2.java

```diff
@@ -0,0 +1,105 @@
+package com.google.jenkins.plugins.computeengine.client;
```

Comment:
adding a new class `ComputeClientV2` for missing feature in graphite that were needed here